### PR TITLE
Eslint instead of tslint (commons, contracts, react, wallet, web3)

### PR DESCRIPTION
**Pull request by Marik-D**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [x] Change is small and easy to review (please split big changes into multiple PRs)
- [x] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [x] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-react/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "prebuild": "yarn clean",
     "start": "webpack-dev-server --mode development",
-    "lint": "tslint -p tsconfig.json \"{src,test}/**/*.{ts,tsx}\"",
+    "lint": "eslint '{src,test}/**/*.{js,ts,tsx}'",
```
Comment (xitronix) [inline universal-login-react/package.json]: Why we need js files in universal-login/react?

File: universal-login-commons/package.json [hunk 2]
```diff
@@ -11,23 +11,25 @@
     "clean": "rimraf dist",
     "test": "yarn build && yarn test:nobuild",
     "test:nobuild": "mocha -t 10000 \"dist/test/**/*.js\"",
-    "lint": "tslint -p tsconfig.json \"{lib,test}/**/*.{ts,js}\""
+    "lint": "eslint '{lib,test}/**/*.{js,ts}'"
```
Comment (xitronix) [inline universal-login-commons/package.json]: Do we need j here?

**Comment by rzadp** [comment]
Why do we want eslint instead of tslint?

**Comment by Marik-D** [comment]
TS-lint is getting deprecated, eslint is more stable, has more features, and will soon supersede tslint anyway

File: universal-login-react/src/main.tsx [hunk 3]
```diff
@@ -8,9 +8,9 @@ const services = createServices(config);
...
-    <ServiceContext.Provider value={services}>
-      <App/>
-    </ServiceContext.Provider>
-  ),
-  document.getElementById('app')
+  <ServiceContext.Provider value={services}>
+    <App/>
+  </ServiceContext.Provider>
+),
+document.getElementById('app'),
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/main.tsx]: Why?

File: universal-login-react/src/ui/commons/GasPrice/GasPrice.tsx [hunk 4]
```diff
@@ -24,12 +24,15 @@ interface GasPriceProps {
...
-  }
+
+  useAsyncEffect(async () => {
+    if (isDeployed) {
+      ensureNotNull(deployedWallet, Error, 'Missing parameter: deployedWallet');
+      return deployedWallet!.subscribeToBalances(setTokenDetailsWithBalance);
+    } else {
+      ensureNotNull(sdk, Error, 'Missing parameter: sdk');
+    }
+  }, [isDeployed, deployedWallet]);
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/GasPrice/GasPrice.tsx]: 👍

File: universal-login-wallet/src/ui/react/Modals/Transfer/ModalTransfer.tsx [hunk 5]
```diff
@@ -10,7 +10,10 @@ const ModalTransfer = () => {
   const [modal, setModal] = useState('transferAmount');
 
   const {walletService, sdk} = useServices();
-  const [transferDetails, setTransferDetails] = useState({transferToken: ETHER_NATIVE_TOKEN.address} as TransferDetails);
+  // FIXME: This cast looks like a bug
+  const [transferDetails, setTransferDetails] = useState(
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/src/ui/react/Modals/Transfer/ModalTransfer.tsx]: It looks like the default value of `transferDetails`. We don't want to have a default amount, but the default token is ok. I don't see a bug here. We can remove this comment and add a task to look at this. Maybe we can move this default value to the bottom? Let's discuss later.

**Pull request merged**